httpapi/httpapi.go

```diff
@@ -23,9 +29,21 @@ func NewServeMux(nodes []*node.HydraNode) *http.ServeMux {
 				Addrs: n.Host.Addrs(),
 			})
 		}
-	})
+	}
+}
 
-	return mux
+// "/records/fetch" Receive a record and fetch it from the network, if available
+func recordFetchHandler(nodes []*node.HydraNode) func(w http.ResponseWriter, r *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		// TODO
+	}
+}
+
+// "/records/list" Receive a record and fetch it from the network, if available
+func recordListHandler(nodes []*node.HydraNode) func(w http.ResponseWriter, r *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		// TODO
```

Comment:
Looking how to do this one out https://discuss.libp2p.io/t/list-provider-records/450